linear_algebra/src/Rank_of_Matrix.py

```diff
@@ -0,0 +1,53 @@
+"""This is a python program that calculates the rank of a matrix"""
+""" BY - RUDRANSH BHARDWAJ"""
+
+
+def swapRows(a, row1, row2):
```

Comment:
To resolve `ruff` errors, you should rename these functions to use lowercase letters. For example, you could rename "swapRows" to "swap_rows", "Row_Transformation" to "row_transformation", and "MatrixRank" to "matrix_rank".

Reply:
Ok let me try